application/src/pages/create-product.tsx

```diff
@@ -10,7 +10,7 @@ export default function CreateProduct() {
     const [description, setDescription] = useState("");
     const [raiseAmount, setRaiseAmount] = useState("");
 
-    const { createProduct } = useManifest();
+    const { createProduct, isLoading } = useManifest();
```

Comment:
is that okay that isLoading belongs to useManifest hook? should useManifest need to be awarded about loading?

Reply:
I get it. then in future if isLoading will be used in multiple functions, we will need to create a separate hook related to that I guess.